google/cloud/bigquery_storage_v1/async_client.py

```diff
@@ -14,23 +14,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Wrappers for protocol buffer enum types."""
+"""Parent async client for calling the Cloud BigQuery Storage API.
 
-import enum
+This is the base from which all interactions with the API occur.
+"""
 
+from __future__ import absolute_import
+
+
+from google.cloud.bigquery import storage_v1
 
-class DataFormat(enum.IntEnum):
-    """
-    Data format for input or output data.
-
-    Attributes:
-      DATA_FORMAT_UNSPECIFIED (int)
-      AVRO (int): Avro is a standard open source row based file format.
-      See https://avro.apache.org/ for more details.
-      ARROW (int): Arrow is a standard open source column-based message format.
-      See https://arrow.apache.org/ for more details.
-    """
 
-    DATA_FORMAT_UNSPECIFIED = 0
-    AVRO = 1
-    ARROW = 2
+_SCOPES = (
+    "https://www.googleapis.com/auth/bigquery",
+    "https://www.googleapis.com/auth/cloud-platform",
+)
+
+
+class BigQueryReadAsyncClient(storage_v1.BigQueryReadAsyncClient):
```

Comment:
On second-thought, maybe this is a task for me. We're going to need some design work to handle the stream resumption logic for the async client. Let's hold off on adding this module for now. Folks can use the generated async client directly, but let's not expose it at the top `bigquery_storage_v1` or `google.cloud.bigquery.storage` level yet.

Reply:
Fine then, removed the async client from "main" import paths for now.